apps/meteor/app/message-snippet/client/snippetMessage.js

```diff
@@ -1,20 +1,12 @@
 import { Meteor } from 'meteor/meteor';
```

Comment:
What this Snippet message does? I saw that you removed the settings, is really necessary?

Reply:
looks like it's updating the Message collection to set the snippeted field as true, but I don't see it being used anywhere, could you confirm if we use it? Otherwise I'll remove it then